x-pack/plugins/maps/public/components/layer_panel/join_editor/resources/join_expression.js

```diff
@@ -222,3 +249,9 @@ JoinExpression.propTypes = {
   rightFields: PropTypes.object, // indexPattern.fields IndexedArray object
   onRightFieldChange: PropTypes.func.isRequired,
 };
+
+function getSelectFieldPlaceholder() {
```

Comment:
This needs to return something. Right now there is not return so the placeholder is blank. Also, the default value should be `Select field`

<img width="353" alt="screen shot 2019-03-06 at 12 03 05 pm" src="https://user-images.githubusercontent.com/373691/53906320-e1dfcc80-4007-11e9-8013-80bc17017edc.png">